Obsidian_Runtime/src/main/yul_templates/object.mustache

```diff
@@ -82,16 +87,23 @@ object "{{creationObject}}" {
                 mstore(pos, cleanup_t_uint256(value))
             }
 
+            function revert_forward_1() {
```

Comment:
What does this do? A comment would help.

Reply:
It reverts but also copies out the return data from memory. I think it's to aid debugging; I found it in the generated yul from solc and didn't want to mess with it because I didn't know how important it was. I'll add a comment.